### PR TITLE
Chunter: Delete message

### PR DESCRIPTION
**Pull request by denis-bunakalya**
https://github.com/hcengineering/anticrm/issues/1300

### CODE REVIEW

File: plugins/chunter-resources/src/components/Message.svelte [hunk 1]
```diff
@@ -59,6 +60,23 @@
         action: chunter.actionImpl.SubscribeMessage
       } as Action)
 
+  async function deleteMessage () {
+    if (space) {
+      await client.removeDoc(chunter.class.Message, space, message._id)
+      const loc = getCurrentLocation()
+
+      if (loc.path.length > 3) {
+        loc.path.length = 3
```
Comment (BykhovDenis) [inline plugins/chunter-resources/src/components/Message.svelte:67]: Why we need change current url?

File: plugins/chunter-resources/src/components/Message.svelte [hunk 2]
```diff
@@ -59,6 +60,23 @@
         action: chunter.actionImpl.SubscribeMessage
       } as Action)
 
+  async function deleteMessage () {
+    if (space) {
+      await client.removeDoc(chunter.class.Message, space, message._id)
```
Comment (BykhovDenis) [inline plugins/chunter-resources/src/components/Message.svelte]: You have space in message, you don't need it. Also you should use `await client.remove(message)`

File: plugins/chunter-resources/src/components/Message.svelte [hunk 3]
```diff
@@ -37,6 +37,7 @@
   export let message: WithLookup<Message>
   export let employees: Map<Ref<Employee>, Employee>
   export let thread: boolean = false
+  export let space: Ref<Space> | undefined
```
Comment (BykhovDenis) [inline plugins/chunter-resources/src/components/Message.svelte]: You don't need it

File: plugins/chunter-resources/src/components/Message.svelte (hunk 1 above)
Comment (BykhovDenis) [reply plugins/chunter-resources/src/components/Message.svelte:67]: You should check it before close thread `if (loc.path[3] === message._id)`

**Review by BykhovDenis — CHANGES_REQUESTED** [review]

File: plugins/chunter-resources/src/components/Message.svelte (hunk 1 above)
Comment (denis-bunakalya) [reply plugins/chunter-resources/src/components/Message.svelte:67]: Done

File: plugins/chunter-resources/src/components/Message.svelte (hunk 2 above)
Comment (denis-bunakalya) [reply plugins/chunter-resources/src/components/Message.svelte]: Done

File: plugins/chunter-resources/src/components/Message.svelte (hunk 3 above)
Comment (denis-bunakalya) [reply plugins/chunter-resources/src/components/Message.svelte]: Removed

**Review by BykhovDenis — APPROVED** [review]

**Pull request merged**